### PR TITLE
Auto publish NPM package on merge to master

### PR DESCRIPTION
**Pull request by cpjolicoeur**

### CODE REVIEW

File: scripts/npm_version_check.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  echo 0
+}
+
+function versionCheck() {
+  echo "Checking pacakge version bump"
+
+  newVersion=$( grep '"version":' ${packageMeta} | cut -d\" -f4 )
+  echo "  current version:    ${newVersion}"
+
+  npmVersion=$( npm show @mojotech/amica-common version)
```
Comment (sethkrasnianski) [inline scripts/npm_version_check.sh]: Looks like we need to update the name here

Comment (cpjolicoeur) [reply scripts/npm_version_check.sh]: slow down charlie, I'm already on it

Comment (cpjolicoeur) [reply scripts/npm_version_check.sh]: fixed and green

Comment (sethkrasnianski) [reply scripts/npm_version_check.sh]: I think you meant deputy ;)

Comment (cpjolicoeur) [reply scripts/npm_version_check.sh]: https://www.dictionary.com/browse/synonym

**Pull request merged**